### PR TITLE
fix multimedia map app id, don't render menu when returning form resp…

### PR DESCRIPTION
**Pull request by wpride**
…onse @benrudolph

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ FormplayerFrontend.getCurrentRoute = function () {
  * The actual mapping is contained in the app Couch document
  */
 FormplayerFrontend.reqres.setHandler('resourceMap', function (resource_path, app_id) {
+    debugger;
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js]: Unexpected 'debugger' statement. (no-debugger)

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_controller.js [hunk 2]
```diff
@@ -33,11 +33,12 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
             };
             if (menuResponse.type === "commands") {
                 menuListView = new MenuList.MenuListView(menuData);
+                FormplayerFrontend.regions.main.show(menuListView.render());
             }
             else if (menuResponse.type === "entities") {
                 menuListView = new MenuList.CaseListView(menuData);
+                FormplayerFrontend.regions.main.show(menuListView.render());
             }
-            FormplayerFrontend.regions.main.show(menuListView.render());
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_controller.js]: don't need to change this

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js]: debug

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_controller.js (hunk 2 above)
Comment (wpride) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_controller.js]: not related but we do - the form response will also come through here which is not displayed in the menu region

**Pull request merged**